src/app/playground/page.tsx

```diff
@@ -0,0 +1,87 @@
+"use client"
```

Comment:
Loading the wasm module does not work, and the `useEffect` alsoe blocks this on the server.